packages/gestalt/package.json

```diff
@@ -31,5 +39,15 @@
     "last 2 versions",
     "not IE < 11",
     "not <1%"
-  ]
+  ],
+  "keywords": [
+    "react",
+    "react component",
+    "pinterest",
+    "ui library"
```

Comment:
Should we add "design system" here?

Reply:
Definitely - added in https://github.com/pinterest/gestalt/pull/1271